### ISSUE TITLE
!Critical. Game crashes on loading world   

### ISSUE THREAD

**Issue by Zizes** [FTBTeam/FTB-Modpack-Issues#6352]
### Modpack

FTB Presents Direwolf20 1.21

### Modpack version

1.7.0

### Launcher

FTB App

### Has the pack been modified

No

### Log Files

[console.log](https://github.com/user-attachments/files/18049411/console.log)
[debug.log](https://github.com/user-attachments/files/18049412/debug.log)
[console.log](https://github.com/user-attachments/files/18049414/console.log)
[latest.log](https://github.com/user-attachments/files/18049415/latest.log)
[rocket-client.log](https://github.com/user-attachments/files/18049416/rocket-client.log)
[crash-2024-12-07_20.07.46-server.txt](https://github.com/user-attachments/files/18052778/crash-2024-12-07_20.07.46-server.txt)

### Describe the bug

I upgraded version from 1.5.0 to 1.7.0. After that when I am trying to open my world, game crashes without any warning window. 

### Steps to reproduce

If only I know... Better to look what is in the logs.

1. upgrade version from 1.5.0 to 1.7.0
2. Try to load already existed world. 

If I understand correctly, problem is with nickel dust.

### Expected behaviour

Upgrade version to 1.7.0, Launch existed world without any bug, Play game.

**Comment by ArtpokeBr** [comment]
You mean... Version 1.7.0 right? please confirm the version that you're in.

Also in your crash log, it states that its a problem somewhat related to JAOPCA, Nickel Dust is Missing, wich is.. weird 🤔

**Comment by Zizes** [comment]
Yes. Version is 1.7.0. I tried with 1.6.0 and had same issue. Only 1.5.0 works.

**Comment by Garrythelawless1** [comment]
I can confirm this issue.  I have not been able to update to the latest build since 1.5 came out.   I will upload logs in a few

**Comment by Garrythelawless1** [comment]
I can confirm this issue.  I have not been able to update to the latest build since 1.5 came out.

Log Files
[console.log](https://github.com/user-attachments/files/18135828/console.log)
[debug.log](https://github.com/user-attachments/files/18135829/debug.log)
[latest.log](https://github.com/user-attachments/files/18135837/latest.log)
[rocket-client.log](https://github.com/user-attachments/files/18135838/rocket-client.log)
[2024-12-13-6.log.gz](https://github.com/user-attachments/files/18135849/2024-12-13-6.log.gz)

**Comment by Gaz492** [comment]
Apologies for the lack of replies, is this still an issue for you?
